phidgets_imu/src/imu_ros_i.cpp

```diff
@@ -209,8 +210,13 @@ bool ImuRosI::calibrateService(std_srvs::Empty::Request &req,
 
 void ImuRosI::calibrate()
 {
-    ROS_INFO("Calibrating IMU...");
+    ROS_INFO(
+        "Calibrating IMU, this takes around 2 seconds to finish. "
+        "Make sure that the device is not moved during this time.");
```

Comment:
I think INFO is okay here? Or should I make it a warning as in the previous commit?

Reply:
No, info is better.